app/controllers/protocols_controller.rb

```diff
@@ -1120,9 +1108,9 @@ def check_copy_to_repository_permissions
     @protocol = Protocol.find_by_id(params[:id])
     @my_module = @protocol.my_module
 
-    if @my_module.blank? || !can_copy_protocol_to_repository(@my_module)
-      render_403
-    end
+    render_403 unless @my_module.present? &&
```

Comment:
Is this ok?

Reply:
Yes, because you the protocol needs to be in the module in order to be copied to the repository.

Reply:
:+1: